### PR TITLE
Add CODEOWNERS File

### LINKED ISSUE #163: Create CODEOWNERS File
Create a `CODEOWNERS` file to have people automatically requested for reviews on PRs

### PR DESCRIPTION
**Pull request by Darth-Raazi**
Created a `CODEOWNERS` file and added myself, Jack and Kavin as owners

Closes #163

- - -
This change is [Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/164)

### CODE REVIEW

**Review by QuantumManiac — APPROVED** [review]
:lgtm:, left some non-blocking reviews. Be sure to verify functionality once this is merged and Code Owners are up and running. 

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/164#-:-NhyPHO11qKlhN_FBbQC:buvzk5h)* status: all files reviewed, 1 unresolved discussion (waiting on @BluCodeGH, @Darth-Raazi, and @KavinSatheeskumar)

___
*[`.github/CODEOWNERS` line 4 at r1](https://reviewable.io/reviews/waterloo-rocketry/omnibus/164#-NhyOOMtC8b-iDxNTrSW:-NhyOOMtC8b-iDxNTrSX:b-hvpbi0) ([raw file](https://github.com/waterloo-rocketry/omnibus/blob/f4fc01da40637f9a373911d60f556dd9a4de09af/.github/CODEOWNERS#L4)):*
> ```CODEOWNERS
> # These people will also be automatically requested for review on PRs
> 
> * @Darth-Raazi @BluCodeGH @KavinSatheeskumar
> ```

Add me :pleading_face: 

Two points:
- Is there any way to list the owners of a file as a list (i.e. one user per line)? I feel that having all the names on a single line might get messy
- Assuming that you intend on the above users beings the "default" reviewers for pretty much any software project, thoughts on creating a `default-reviewers` team and adding that instead of the above individually? It might be nice to allow for easy distinguishing between actual contributors to the project and default reviewers who show up on pretty much every `CODEOWNERS` file.

**Review by Darth-Raazi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/164)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @BluCodeGH, @KavinSatheeskumar, and @QuantumManiac)

___
*[`.github/CODEOWNERS` line 4 at r1](https://reviewable.io/reviews/waterloo-rocketry/omnibus/164#-NhyOOMtC8b-iDxNTrSW:-NiCHf6cBxvie6ZVs6NF:b-bjqk00) ([raw file](https://github.com/waterloo-rocketry/omnibus/blob/f4fc01da40637f9a373911d60f556dd9a4de09af/.github/CODEOWNERS#L4)):*
<details><summary><i>Previously, QuantumManiac (Chamath Wijesekera) wrote…</i></summary><blockquote>

Add me :pleading_face: 

Two points:
- Is there any way to list the owners of a file as a list (i.e. one user per line)? I feel that having all the names on a single line might get messy
- Assuming that you intend on the above users beings the "default" reviewers for pretty much any software project, thoughts on creating a `default-reviewers` team and adding that instead of the above individually? It might be nice to allow for easy distinguishing between actual contributors to the project and default reviewers who show up on pretty much every `CODEOWNERS` file.
</blockquote></details>

1. I think it's fine on one line for now, maybe we can revisit if there's 10+ codeowners
2. Done

**Review by QuantumManiac — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/164#-:-NiFj6PZ3cvwLqXPNVUs:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @BluCodeGH and @KavinSatheeskumar)

**Pull request merged**
